### ISSUE TITLE
garageplay.tw [Anti-Adblock]

### ISSUE THREAD

**Issue by jdck** [uBlockOrigin/uAssets#12313]
### Prerequisites

- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues) to avoid opening a duplicate issue
- [X] I am running the latest version of uBlock Origin

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### URL(s) where the issue occurs

`https://garageplay.tw/anipass/AnipassVideo/Player/2772`
`https://garageplay.tw/anipass/AnipassVideo/Player/2773`

### Describe the issue

The site's anime has anti-adblockers, which will cause the video not to play.

### Screenshot(s)

![image](https://user-images.githubusercontent.com/16664309/158943794-52815db7-274e-4f51-95b6-cde6e5138d70.png)

### uBlock Origin version

1.41.8

### Browser name and version

Google Chrome 99.0 (64-bit)

### Settings

- Default

**Comment by mapx-** [comment]
go into dashboard, support tab, post here the `Troubleshooting Information`

**Comment by jdck** [comment]
> go into dashboard, support tab, post here the `Troubleshooting Information`

uBlock Origin: 1.41.8
Chromium: 99
filterset (summary): 
  network: 97239
  cosmetic: 51697
  scriptlet: 16661
  html: 0
listset (total-discarded, last updated): 
  added: 
    https://raw.githubusercontent.com/cjx82630/cjxlist/master/cjxlist.txt: 516-3, 5d.14h.2m
  default: 
    user-filters: 0-0, never
    CHN-0: 27971-87, 1d.10h.47m
    easylist: 64273-410, 1d.10h.46m
    easyprivacy: 26702-147, 1d.10h.44m
    plowe-0: 3671-807, 1d.20h.30m
    ublock-abuse: 72-0, 1d.10h.45m
    ublock-badware: 3972-84, 1d.10h.51m
    ublock-filters: 30355-156, 1d.10h.50m
    ublock-privacy: 200-2, 1d.10h.49m
    ublock-unbreak: 1739-42, 1d.10h.48m
    urlhaus-1: 8023-0, 3m
filterset (user): [empty]
trustedset: 
  added: [array of 1 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 321 ms (selfie)
  maxAssetCacheWait: 348 ms

**Comment by stephenhawk8054** [comment]
Is this one also fixed by your solution in issue [https://github.com/uBlockOrigin/uAssets/issues/12312](https://github.com/uBlockOrigin/uAssets/issues/12312)?

**Comment by jdck** [comment]
> Is this one also fixed by your solution in issue #12312?

Nope

**Comment by stephenhawk8054** [comment]
> > Is this one also fixed by your solution in issue #12312?
> 
> Nope

I just checked it, and it's because of these 2 filters

```
##.text-ad
##.textAd
```

in your custom filter list: `https://raw.githubusercontent.com/cjx82630/cjxlist/master/cjxlist.txt`.

You can report to the maintainer of that list, or add these exceptions yourself

```
garageplay.tw#@#.text-ad
garageplay.tw#@#.textAd
```

Anyway, please test correctly with your ticks

>  ### I tried to reproduce the issue when...
> 
> - [X] uBO with default lists/settings
